src/main/java/org/broadinstitute/hellbender/dev/pipelines/bqsr/ApplyBQSRTransform.java

```diff
@@ -0,0 +1,79 @@
+package org.broadinstitute.hellbender.dev.pipelines.bqsr;
+
+import com.google.api.services.genomics.model.Read;
+import com.google.cloud.dataflow.sdk.transforms.DoFn;
+import com.google.cloud.dataflow.sdk.transforms.PTransform;
+import com.google.cloud.dataflow.sdk.transforms.ParDo;
+import com.google.cloud.dataflow.sdk.transforms.View;
+import com.google.cloud.dataflow.sdk.values.PCollection;
+import com.google.cloud.dataflow.sdk.values.PCollectionView;
+import com.google.cloud.genomics.dataflow.readers.bam.ReadConverter;
+import com.google.cloud.genomics.gatk.common.GenomicsConverter;
+import htsjdk.samtools.SAMException;
+import htsjdk.samtools.SAMFileHeader;
+import htsjdk.samtools.SAMRecord;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.broadinstitute.hellbender.tools.ApplyBQSRArgumentCollection;
+import org.broadinstitute.hellbender.transformers.BQSRReadTransformer;
+
+/**
+ * This transforms applies BQSR to the input reads.
+ */
+public final class ApplyBQSRTransform extends PTransform<PCollection<Read>, PCollection<Read>> {
+
+    private final SAMFileHeader header;
+    private final PCollectionView<BaseRecalOutput> recalView;
+    private final ApplyBQSRArgumentCollection options;
+
+    /**
+     * @param header The SAM header that corresponds to the reads you're going to pass as input.
+     * @param phaseOne the output from BaseRecalibration, with a single BaseRecalOutput object.
+     * @param options the recalibration options
+     */
+    public ApplyBQSRTransform(SAMFileHeader header, PCollection<BaseRecalOutput> phaseOne, ApplyBQSRArgumentCollection options) {
```

Comment:
It seems like our convention is to pass around the SAMFileHeader wrapped as a `PCollectionView`.  I'm not sure I quite understand why.  I think it's to safeguard against the future where it might need to be side input.